pages/common/brew.md

```diff
@@ -0,0 +1,32 @@
+# brew
+
+> Package manager for macOS and Linux.
+> More information: <https://brew.sh>.
+
+- Search for available formulae (i.e. packages) and casks (i.e. native packages):
+
+`brew search {{text}}`
+
+- Install the latest stable version of a formula or cask (use `--devel` for development versions):
+
+`brew install {{formula}}`
+
+- List all installed formulae and casks:
+
+`brew list`
+
+- Upgrade an installed formula or cask (if none is given, all installed formulae/casks are upgraded):
+
+`brew upgrade {{formula}}`
+
+- Fetch the newest version of Homebrew and of all formulae and casks from GitHub:
```

Comment:
This confused me for just a second, because it can sound like it downloads them from the tools' respective GitHub repos rather than the main Homebrew repository.  Just a thought.
```suggestion
- Fetch the newest version of Homebrew and of all formulae and casks from the Homebrew source repository:
```

Reply:
Or this ?
```suggestion
- Fetch the newest version of Homebrew and of all formulae and casks:
```

Reply:
That's fine, but it might be nice to know where they're coming from.  Is there anything about my suggestion that you don't like? (💯 fine if there is)

Reply:
I find your suggestion OK. :ok_hand: